IPython/core/display.py

```diff
@@ -852,18 +852,24 @@ def _data_and_metadata(self):
     def _repr_json_(self):
         return self._data_and_metadata()
 
-_css_t = """$("head").append($("<link/>").attr({
-  rel:  "stylesheet",
-  type: "text/css",
-  href: "%s"
-}));
+_css_t = """var link = document.createElement("link");
+	link.ref = "stylesheet";
+	link.type = "text/css";
+	link.href = "%s";
+	document.head.appendChild(link);
 """
 
-_lib_t1 = """$.getScript("%s", function () {
-"""
-_lib_t2 = """});
+_lib_t1 = """var script = document.createElement("script");
+	script.type = "text/javascript";
+	script.src = "%s";
+	script.onload = script.onreadystatechange = function() {
+    if (!this.readyState || this.readyState == 'complete') {
```

Comment:
Is there a chance that this will never run the script, where `onload` or `onreadystatechange` are called yet `this.readyState` is not complete?